packages/uicore/src/components/ExpressionDropdown/ExpressionDropdown.css

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+.dropdown {
+  padding: 10px 0px 3px 5px !important;
+}
+
+.overflowList {
+  max-width: 600px;
+}
+
+.paddingLeft {
+  padding-left: 5px !important;
+}
+
+.visibleItemTextHeader {
+  padding-left: 5px;
+  font-weight: 900;
+}
+
+.visibleItemText {
+  color: var(--blue-800) !important;
```

Comment:
primary-7

Reply:
Will update this once UX updates on the final color